plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -181,7 +187,7 @@ public TableInfo getCachedTable(Duration viewExpiration, TableInfo remoteTableId
 
     public String getProjectId()
     {
-        return bigQuery.getOptions().getProjectId();
+        return projectId.orElse(bigQuery.getOptions().getProjectId());
```

Comment:
Now, the returned value of `BigQueryMetadata.getProjectId` looks the same as `BigQueryClient.getProjectId.` We can merge the method in BigQueryMetadata into BigQueryClient class.

Reply:
Moved the logic in `BigQueryMetadata.getProjectId` into `BigQueryClient.getProjectId`, removed `BigQueryMetadata.getProjectId` and replaced all calls in `BigQueryMetadata` of `getProjectId(client)` with `client.getProjectId()`.